### ISSUE TITLE
Moving /Applications/Soundflower/ into a different directory causes 30 second delay when switching users/logging out of user accounts

### ISSUE THREAD

**Issue by GoogleCodeExporter** [mattscotthudson/soundflower#250]
```
What steps will reproduce the problem?
1. Move /Applications/Soundflower/ into a subdirectory (e.g: 
/Applications/FolderedApps/Soundflower/)

2. Fast User Switch to user 2
Expected behavior: Instantly switches to user
Actual behavior: Takes several seconds to switch to user

3. Log out of user 2
Expected behavior: Instantly logs out and goes to system log in screen
Actual behavior: 30 second delay on gray screen before going to system log in 
screen

What is the expected output? What do you see instead?
Basically, it appears that moving the Soundflower folder into a different 
subfolder causes problems when switching users and logging out; presumably 
/Applications/Soundflower/ is hard-coded somewhere...

What version of the product are you using? On what operating system?
Soundflower 1.6.6b
Mac OS X 10.9.1

Please provide any additional information below.


```

Original issue reported on code.google.com by `Toasty...@gmail.com` on 27 Jan 2014 at 9:49

**Comment by GoogleCodeExporter** [comment]
```
Hm. Actually, even after leaving Soundflower/ in /Applications, I eventually 
started getting the 30 second delay again. After uninstalling Soundflower 
altogether, it's gone. So there's something broken when launching.
```

Original comment by `Toasty...@gmail.com` on 9 Feb 2014 at 5:03
